uiowa/uiowa
# Remove uiowa_people part 3

Runs previous update hooks to remove uiowa_people from the sites.

careers.stage is an example of where this is still an issue. https://github.com/orgs/uiowa/projects/1#card-65146215

## To Test

- `git checkout main`
- `composer install && blt frontend`
- change the drush -> remote to `careers.test`
- `blt ds --site=careers.uiowa.edu`
This should fail!

- `git checkout uiowa-people-removal-part-3`
- `blt ds --site=careers.uiowa.edu`
This should run smoothly!
